### ISSUE TITLE
Need to write a page for frankenfirmwares cannot be web-booted with rxTools 2.5.2

### ISSUE THREAD

**Issue by MelonGx** [Plailect/Guide#131]
Such as 7.1.0-4J.
Otherwise the noobs who have this frankenfirmware will stuck at 9.2.0 Update's Step 26.

BTW, for my information collection, 4.3.0-4J can be web-booted with rxTools 2.5.2 but 7.1.0-4J cannot.
That means, there are some other things (maybe NATIVE_FIRM?) out of the Internet Browser can affect the rxTools 2.X web entrypoint.

If a frankenfirmware rxTools 2.X web entrypoint does not work, then we have only GW web entrypoint works.
But unfortunately, there are no GW web-boot supported CFWs with Sig-Patch-Only mode access.
For these frankenfirmwares, we have the following method to get them work:
- (Method 1) Boot by GW, CIA upgrading emuNAND to 9.2.0-20, GW downgrading 9.2-Emu to 4.X, convert 4.X-Emu to RedNAND, write 4.X-Red to SysNAND **(Of course backup your original before writing 4.X-Red!)**
- (Method 2) Cart-upgrade it to a HBL bootable 9.0-10.7 -4, use HBL to boot SafeSysupdater to downgrade it to 9.2.0-20 (Someone tested that 9.2.0-4U froze at booting HBL but 9.9.0-4U worked)

**Comment by MelonGx** [comment]
BTW, this wiki also lacks -0 frankenfirmwares' instruction.

Since -0 has no browser, all the web-boot entrypoints (and browserhax) does not work.

A general method working for every -0 consoles is
- Cart-upgrade to 9.9, boot HBL with anything other than Browserhax, then SafeSysupdater-downgrade to 9.2.0-20

For specifying N3DS 8.1.0-0J / 8.1.0-0E, we still have private SOAP server method. But it's more complicated than current version - 9.9 - 9.2.

**Comment by MelonGx** [comment]
#132 is possibly a prove that 5.1.0-4U also has this problem.

**Comment by Plailect** [comment]
I am still trying to decide exactly how to handle all the different old firmware update methods that exist; I'll need some time before I make any decisions on how this will be structured and written.

**Comment by zfirered** [comment]
4.5 cant boot to decrypt9... 
some of them...

**Comment by MelonGx** [comment]
@zfirered
This wiki introduces booting Decrypt9 with GW web entrypoint & using Launcher.dat-format Decrypt9.
It should be no problem since GW web entrypoint had full support of 4.0-9.2 & every browser <= v4096.

**Comment by dark-samus** [comment]
@Plailect there is no reason to use rxTools, you can use another CFW that meets the needs better... also, the D9 launcher.dat entrypoint is depreciated... D9 can launch from the browser using cakehax. For systems that don't have browser, there should be a tutorial on using a flashcart to install the appropriate MSET ROP onto the console, there should also be a note saying that certain firmwares (browserless 5.x or 7.x-8.1) cannot be updated using sysupdater and that they'll need to update with a cart, then get some sort of entry into the system (OoThax, ninjhax 2.x, etc.) from there to either properly update their system (to get a browser) or, if they went higher than 9.2 they need to downgrade.

**Comment by MelonGx** [comment]
In a word, Frankenfirmwares -> 9.3-10.7 -> 9.2 is the universal solution.

5.1.0-4 cannot boot Devmode? Upgrade to 9.9 then DG to 9.2.
Stuck at 7.2.0-0? Upgrade to 9.9 then DG to 9.2.
Don't know how to use NTR + Private Server for N3DS 8.1.0-0J? Upgrade to 9.9 then DG to 9.2.
9.2.0-4 freezes at booting HBL? Upgrade to 9.9 then DG to 9.2.0-20.

**Comment by arromdee** [comment]
@MelonGx: Although it's true that updating to 9.9 and downgrading works for everyone, that can't be the first choice.  Upgrading to 9.9 and downgrading may require using Cubic Ninja/.OOT/PSMD to downgrade, and a way to get the save game onto those last two.  Not everyone has those.  So that method shoul;d only be a last resort, after the user tries the other methods that don't need anything extra.

**Comment by MelonGx** [comment]
New report:
Proper firmware 5.1.0-11U failed booting rxTools 2.X Devmode either.
http://gbatemp.net/threads/problems-trying-to-update-to-9-2.428292/

It seems that rxTools 2.X Devmode only works on main version 4.X or 6.X regardless whether it's frankenfirmware or not.

**Comment by MelonGx** [comment]
Conclusion of all the specific firmwares above which don't work with this guide:
- 5.X 7.X 8.X  - Can't be booted by rxTools 2.5.2 Devmode, but can be booted by rxTools 3.X Pasta Mode (to be noticed that rxTools 3.X doesn't support -2 to -6 web-boot, which rxTools 2.5.2 supports)
- 5.X with -2 to -6 - Neither of rxTools 2.5.2 Devmode, MSET, rxTools 3.X web-boot access, but can be cartridge upgraded to 6.X with -2 to -6 to continue using this guide
- 7.X 8.X with -2 to -6 - Neither of rxTools 2.5.2 Devmode, MSET, rxTools 3.X web-boot access, both Create EmuNAND 9.2 by GW -> DG EmuNAND -> Convert EmuNAND to RedNAND -> Inject RedNAND to SysNAND & upgrade to 9.3 - 10.7 then DG to 9.2 work.
- 7.X 8.X with -0 - Neither browser nor MSET access, only upgrade to 9.3 - 10.7 then DG to 9.2 works
- N3DS 8.1.0-0J - Neither browser nor MSET access, but has NTR access, both NTR + 9.0 - 9.2 private server & upgrade to 9.3 - 10.7 then DG to 9.2 work
- 9.2.0 with specific suffix - It should be available to boot HBL but actually it doesn't, only upgrade to 9.3 - 10.7 then DG works

**Comment by arromdee** [comment]
> 9.0 - 9.2 with specific suffix - It should be available to boot HBL but actually it doesn't, only available with upgrade to 9.3 - 10.7 then DG to 9.2.0-20

Can that ever happen?  

Because there is no useful browser, upgrading and downgrading will only work if you downgrade using CN/OOT/PSMD.  If you have one of those, you don't need to upgrade and downgrade because you can use it to get into homebrew directly.

**Comment by arromdee** [comment]
Also, <=4.x can be cartridge updated to 4.x and you can use mset.

And if you started on 2,1, get the OTP first.  (Is there any way for someone on 1.x or 2.0 to get it aside from using a cart to upgrade to 2.1?)  Believe it or not, there have been cases of people who started with 2.1.

**Comment by arromdee** [comment]
>7.X 8.X with -2 to -6 - Neither of rxTools 2.5.2 Devmode, MSET, rxTools 3.X web-boot access work, available with (1) Create EmuNAND 9.2 by GW -> DG EmuNAND -> Convert EmuNAND to RedNAND -> Inject RedNAND to SysNAND (2) Upgrade to 9.3 - 10.7 then DG to 9.2

(3) If (1) is possible, that means you can boot Gateway.  If you can boot Gateway, you can use the Gateway downgrade to 4.x.  The Gateway downgrade to 4.x will contain a proper browser so you will then be okay.  It is also possible that using the Gateway 4.x downgrade is the solution to some of the other ones, but I don't know which exact versions/browsers can boot Gateway.

**Comment by MelonGx** [comment]
@arromdee

One person told me his 9.2.0-4U cannot boot HBL with Ninjhax. That's why he finally chose 9.2.0-4 -> 9.9.0-4 -> 9.2.0-20.

For factory 1.0-2.2 3DSes, yes, you're right. This wiki should add the description for those machines.

For GWDG on 7.X 8.X -2 to -6, no, it can't. It is tested that GWDG refused performing any downgrade on suffix < -10J / -8E.

**Comment by arromdee** [comment]
I thought that Ninjhax 2 is no longer dependent on the browser.  Was he trying Ninjhax 1 by mistake?

This looks like the kind of thing that someone with a hardmod will have to research so we know for sure.

**Comment by jaxrogers2** [comment]
People with browser versions lower than -7 (such as 2.1.0-6) cannot use Ninjhax 1 or 2. The secondary exploits such as OoT3Dhax would work except those require 9.X to launch HBL.

**Comment by MelonGx** [comment]
I forgot one probability why the guy said 9.2.0-4 -> 9.9.0-4 -> 9.2.0-20 worked:
Since 9.9+ cartridge upgrade pack, Nintendo added a dummy browser.
This may make the *hax treat the SysNAND as a new-browser-installed system.

**Comment by Plailect** [comment]
Added with #336722c

**Comment by coryshaw1** [comment]
Having issues with 5.1.0-10u, can't get rxTools 3.0.0 to boot through the browserhax. Tried rxTools 2.5.1, and successfully can get into rxTools, but can't boot into DevMode. 

It seems that most browserhax don't work on this firmware, the only one that has worked is booting into Decrypt9. Is the only option is to find a cart upgrade, and try from there?

**Comment by coryshaw1** [comment]
Update from above 

5.1.0-10u apparently just doesn't work with rxTools 2.x or 3.x. I ended up using a Pokemon ORAS cart to update to 7.2.0-20u. The guide said to use rxTools 3.x, but was unsuccessful in using browserhax to launch after countless attempts. Tried rxTools 2.5.2 with browserhax exploit to launch, worked first try. 

Got put into DevMode successfully to continue with the 9.2 upgrade. Am now on CFW with latest updates. So should the guide should be updated to say 5.1.0-10u just doesn't work correctly with rxTools? And should the >6u for 7.x.x be updated to rxTools 2.5.2? Maybe there should just be a note at the bottom of the chart of the 9.2.0 upgrade page saying that if rxTools 3.x doesn't work, attempt 2.5.2.
